2020/webkitgtk-npapi-sunsetting.markdown

```diff
@@ -0,0 +1,195 @@
+title: Sunsetting NPAPI plug-in support in WebKitGTK (and WPE)
+date: 2020-09-28 12:00:00
+author: aperez
+tags: igalia, webkit, floss
+
+<ol class="toc">
+  <li><a href="#summary">Summary</a></li>
+  <li><a href="#what-is-npapi">What is NPAPI?</a></li>
+  <li><a href="#what-is-npapi-used-for">What is NPAPI used for?</a></li>
+  <li><a href="#why-are-npapi-plug-ins-being-phased-out">Why are NPAPI plug-ins being phased out?</a></li>
+  <li><a href="#what-are-other-browsers-doing">What are other browsers doing?</a></li>
+  <li><a href="#is-webkitgtk-following-suit">Is WebKitGTK following suit?</a></li>
+</ol>
+
+## Summary
+
+Here's a [tl;dr](https://www.urbandictionary.com/define.php?term=tl%3Bdr)
+list of bullet points:
+
```

Comment:
Perphaps add a quick note about NPAPI. For example:

*NPAPI is and old way to extend the functionality of a web browser. It is time to let it go.*

Reply:
I like this, let's add it.